aip/0008.md

```diff
@@ -174,6 +174,198 @@ When AIPs reference other AIPs, the prosaic text **must** use the format
 link to the relevant AIP. AIP links **may** point to a particular section of
 the AIP if appropriate.
 
+### Static AIP Reference
+
+Most times, you will likely want to use a Context Based Reference when
+providing a link to an AIP (see below). However it's worth noting that there
+may be times when you want to refer to a specific AIP from a specific context,
+regardless of what context you are in.
+
+For example, an organization based AIP may extend AIP-121 by stating that
+everything in the parent AIP-121 is valid, and that the context based AIP-121
+only provides some additional requirements. In this case, the organization
+level AIP-121 would want to provide a link to the domain level AIP-121. If the
+link were a Context Base Reference, the AIP link to AIP-121 would just link
+back to the organization level AIP!
+
+The section below discusses context based references in more detail, and
+provides examples of how these links should be implemented. This section is
+provided for context for these examples, as the implementation examples cannot
+really be shared without consider both types of references together.
+
+### Context Based References
+
+It is not uncommon for AIPs to have relationships or even dependencies on one
+another. For example, it may be logical for some implementations of AIP-131
+(Methods: GET) to reference AIP-121 (Resource Names), as the address used by
+the method should follow the guidelines for resource naming. It may also be
+possible for each of these AIPs to be overridden at an organizational level,
+yet still maintain their relationship one with another.
+
+In such a case, it would be important that the links between these two AIPs
+took into consideration that context under which they were being viewed. For
+example if AIP-131 was being viewed under the organization context, then all
+links to AIP-121 should point to the organizational extension of AIP-121, and
+not the domain level implementation of AIP-121. Likewise, if AIP-121 is being
+viewed at the domain level, then links to AIP-131 should point to the domain
+level implementation of AIP-131, and not to any organizational extensions.
+
+All AIP reference links that need to take context into consideration **must**
```

Comment:
Do we need to do literally anything here? Seems like fork and merge plus static site generation will just handle it since we use `./0100.md` style links.